libs/payments/ui/src/lib/actions/getCart.ts

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use server';
+
+import { app } from '../nestapp/app';
+
+export const getCartAction = async (cartId: string) => {
```

Comment:
Unrelated question to, well, _everything_, but&hellip; if we export `const`s that are basically anonymous functions, does this still give us beautiful breadcrumbs in Sentry after/if things are minified and source mapped? Or would it just be better to export boring old named functions?

Reply:
Unknown. Aren't all NextJS components generally arrow functions?